src/target/adiv5.c

```diff
@@ -721,7 +721,7 @@ void adiv5_dp_init(ADIv5_DP_t *dp)
 
 		/* Check for a valid DPIDR / designer */
 		if (dp->designer_code != 0) {
-			DEBUG_INFO("DP DPIDR 0x%08" PRIx32 " (v%d %srev%d) designer 0x%" PRIx32 " partno 0x%" PRIx32 "\n", dpidr,
+			DEBUG_INFO("DP DPIDR 0x%08" PRIx32 " (v%"PRIx8" %srev%"PRId32") designer 0x%" PRIx16 " partno 0x%" PRIx16 "\n", dpidr,
```

Comment:
PRIx16 and PRIx8 doesn't work on nano newlibc last we tried. These should use %x

Reply:
I've switched it to `%x`, which still compiles on ESP32.